app/boards/shields/cradio/cradio.keymap

```diff
@@ -1,102 +1,103 @@
-/*
- * Copyright (c) 2020 The ZMK Contributors
- *
- * SPDX-License-Identifier: MIT
- */
+// Copyright (c) 2022 The ZMK Contributors
+// SPDX-License-Identifier: MIT
 
 #include <behaviors.dtsi>
 #include <dt-bindings/zmk/keys.h>
 #include <dt-bindings/zmk/bt.h>
 
+// Home row mods macro
+#define HRML(k1,k2,k3,k4) &ht LSHFT k1  &ht LALT k2  &ht LCTRL k3  &ht LGUI k4
+#define HRMR(k1,k2,k3,k4) &ht RGUI k1  &ht RCTRL k2  &ht RALT k3  &ht RSHFT k4
 
-&mt {
-  //  flavor = "tap-preferred";
-   // tapping_term_ms = <200>;
+#define ________ &trans
```

Comment:
TBH, I'm not loving using this... We don't do this everywhere, and it hides what is really going on for these...

Reply:
Good point—overwrite removed as suggested.

Reply:
Yes, exactly.

Reply:
Removed `&trans` substitution macro.